README.md

```diff
@@ -76,11 +112,45 @@ Curation of papers is generally based on the recent publication in top AI confen
 - PLAID [[link]](http://www.plaidplug.com/)
 - ENERTALK [[link]](https://www.nature.com/articles/s41597-019-0212-5)
 
+### Synthetic Datasets
+
+- SHED: [[link]](https://nilm.telecom-paristech.fr/shed/)
+- SmartSim: [[link]](https://github.com/sustainablecomputinglab/smartsim)
+- SynD: [[link]](https://github.com/klemenjak/SynD/)
+
 ## Toolkits
-- NILM-Eval [[codes]](https://github.com/beckel/nilm-eval)
-- NILMTK [[paper]](https://arxiv.org/pdf/1404.3878v1.pdf) [[codes]](https://github.com/nilmtk/nilmtk)
+- NILM-Eval [[pdf]]() [[code]](https://github.com/beckel/nilm-eval)
+- NILMTK [[pdf]](https://arxiv.org/pdf/1404.3878v1.pdf) [[code]](https://github.com/nilmtk/nilmtk)
+  - Recently: NILMTK-Contrib [[pdf]](https://nipunbatra.github.io/papers/batra_buildsys_19.pdf) [[code]](https://github.com/nilmtk/nilmtk-contrib)
+
+## Low-Cost Energy Meters
+
+- YoMoPie: A User-Oriented Energy Monitor to
```

Comment:
I think dealing with energy monitor devices is outside the scope of this repository. Or, if this paper should be included, similar papers for energy monitoring should be included.

Reply:
Well, there are many aspects of NILM. I included our paper because we present an energy monitor that builds on Raspberry Pi. From that follows that Python algorithms can be prototyped on the actual meter. 

Further contributions are:

https://www.researchgate.net/publication/324459175_Design_and_implementation_of_a_low-cost_arduino-based_high-frequency_AC_waveform_meter_board_for_the_raspberry_pi

https://dl.acm.org/citation.cfm?id=3077844

https://ieeexplore.ieee.org/document/7915521

Reply:
It's true that your contribution is important, but it still seems a bit off the topic of NILM. Thank you for your valuable comments.